facts/myredis.py

```diff
@@ -66,7 +66,7 @@ def insert(self, tenantid, serverid, data):
                               is no list or the number of element is not equal to 4.
         """
         ''' we need to check that data is a list and the exact number of
-        element is equal to 3 - Magic Number
+        element is equal to 4 - Magic Number
         '''
         if isinstance(data, list) and len(data) == 4:
```

Comment:
yes, We have everything clear but until it is not implemented, I prefer to preserve the comment